app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java

```diff
@@ -90,6 +85,16 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_description);
 
+        fAuth = FirebaseAuth.getInstance();
+        if (fAuth.getCurrentUser() == null) {
```

Comment:
So you put the authentification first ?

Reply:
Yes exactly, I did that in all classes in such a way that it's the first thing that is done when we create the activity. It a bit more logic and it's avoid other useless objects creation.

Reply:
In fact, this is 1. a good necessary security check not to be left out and 2. this is now done in a logical way! Furthermore, it is now implemented in a consistent way across all our activities! Nice refactor, thanks for this Jean-Luc!